packages/lit-element/src/lib/lit-element.ts

```diff
@@ -216,16 +204,10 @@ export class LitElement extends UpdatingElement {
    */
   protected initialize() {
```

Comment:
I've seen people run into issues because they implemented a method called `initialize`. Would you consider using a different name for this and the static `finalize`? It's a shame for lit-element to take up such common names.

Reply:
Good suggestion. Will handle separately via https://github.com/Polymer/lit-html/issues/1239.